### PR TITLE
Remove defineProperty from DOMTokenList, fix a up a few type annotations...

### PR DESCRIPTION
**Pull request by sayrer**
I /think/ this will pass tests, but I get 

> $ npm test
> dom-shims@0.1.1 test /Users/sayrer/workspace/forks/dom-shims
> karma start --single-run
> Missing local sauce.json with SauceLabs credentials.

when running locally.

### CODE REVIEW

File: shim/CustomEvent.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
 (function (window) {
   'use strict';
-
-  /**
-   * Detect full support
-   */
+  //
```
Comment (necolas) [inline shim/CustomEvent.js]: did the closure compiler complain about the previous comment syntax?

File: shim/Element.mutation.js [hunk 2]
```diff
@@ -72,10 +72,13 @@
     }
   };
 
-  ElementPrototype.remove = function remove() {
+  /**
+   * @param {number=} index
+   */
+  ElementPrototype['remove'] = function remove(index) {
```
Comment (necolas) [inline shim/Element.mutation.js]: why did this need bracket notation?

**Comment by necolas** [comment]
I guess you need your own sauce labs account if you want to run those tests locally? They suggest adding credentials locally using `sauce.json` (see: https://github.com/saucelabs/karma-sauce-example). It's a shame that travis builds using sauce labs also don't work for contributor branches. I'll try it out.

File: shim/CustomEvent.js (hunk 1 above)
Comment (sayrer) [inline shim/CustomEvent.js]: Oh, no, I had it all commented out so changed the comment style to be wrappable.

I'll change it back.

**Comment by necolas** [comment]
Applied in 4f76daf789f1c0355fc9e91bd40c41d3a5db857b. Patch is included in 0.1.2.

**Pull request closed without merging**